homeassistant/components/tplink/switch.py

```diff
@@ -38,9 +39,47 @@ async def async_setup_entry(
     else:
         entities.append(SmartPlugSwitch(device, coordinator))
 
+    entities.append(SmartPlugLedSwitch(device, coordinator))
+
     async_add_entities(entities)
 
 
+class SmartPlugLedSwitch(CoordinatedTPLinkEntity, SwitchEntity):
+    """Representation of switch for the LED of a TPLink Smart Plug."""
+
+    coordinator: TPLinkDataUpdateCoordinator
+
+    def __init__(
+        self, device: SmartDevice, coordinator: TPLinkDataUpdateCoordinator
+    ) -> None:
+        """Initialize the LED switch."""
+        super().__init__(device, coordinator)
+
+        self._attr_entity_category = ENTITY_CATEGORY_CONFIG
```

Comment:
As this doesn't depend on any inputs, this could be done on the class level (i.e., adding `_attr_entity_category = ...` just under the coordinator type hint above.

Reply:
Will update!